docs/table/pandas.rst

```diff
@@ -56,21 +55,18 @@ It is also possible to create a table from a `DataFrame`_::
 
 The conversions to and from ``pandas`` are subject to the following caveats:
 
-* The ``pandas`` `DataFrame`_ structure does not support multidimensional
-  columns, so :class:`~astropy.table.Table` objects with multidimensional
-  columns cannot be converted to `DataFrame`_.
-
-* Masked tables can be converted, but `DataFrame`_ uses ``numpy.nan`` to
-  indicate masked values, so all numerical columns (integer or float) are
-  converted to ``numpy.float`` columns in `DataFrame`_, and string columns with
-  missing values are converted to object columns with ``numpy.nan`` values to
-  indicate missing values. For numerical columns, the conversion therefore does
-  not necessarily round-trip if converting back to an ``astropy`` table,
-  because the distinction between ``numpy.nan`` and masked values is lost, and
-  the different integer columns (for example) will be converted to floating-
-  point.
-
-* Tables with mixin columns can currently not be converted, but this may be
-  implemented in the future.
-
-.. _DataFrame: http://pandas-docs.github.io/pandas-docs-travis/
+* The :class:`~pandas.DataFrame` structure does not support multidimensional
+  columns, so |Table| objects with multidimensional columns cannot be converted
+  to :class:`~pandas.DataFrame`.
+
+* Masked tables can be converted, but :class:`~pandas.DataFrame` uses
```

Comment:
Actually this is no longer a problem for int dtype since #9541 and support for pandas nullable int. So you should update this bit accordingly.

Reply:
I think this is implemented. @taldcroft, resolve conversation?